repos/system_upgrade/el7toel8/libraries/testutils.py

```diff
@@ -24,6 +28,53 @@ def __call__(self, report_fields):
             self.report_fields.update(report.to_dict())
 
 
+# class run_mocked(object):
+
+class logger_mocked(object):
```

Comment:
I have a big doubt about mocking the logger. Unless we're not testing some magic behind leapp implementation of the logger, we could use, already provided, https://docs.pytest.org/en/latest/logging.html#caplog-fixture. 
/cc @pirat89 @vinzenz